### PR TITLE
Add more links to "About this Release" 

### PR DESCRIPTION
**Pull request by sherenewso2**
## Purpose
- Add related links
- Add deprecated features section
- Remove 'scope-based authorization" from navigation (the content in that page has now been moved to a section on another page"

### CODE REVIEW

File: en/docs/get-started/about-this-release.md [hunk 1]
```diff
@@ -1,20 +1,34 @@
...
+
+- Introduced CORS management APIs. For more information, see [CORS management APIs](../../develop/cors-rest-api).
+
+- Upgraded to Open SAML 3
+
+
+## Deprecated Features
+
+- /identity/connect/register API
+- SCIM 1.1 API
```
Comment (GANGANI) [inline en/docs/get-started/about-this-release.md:25]: AFAIK, SCIM 1.1 is not deprecated in this release[1]. We are planned to deprecate it in the future.

[1] [https://is.docs.wso2.com/en/latest/setup/wso2-identity-server-feature-deprecation/](https://is.docs.wso2.com/en/latest/setup/wso2-identity-server-feature-deprecation/)

File: en/docs/get-started/about-this-release.md [hunk 2]
```diff
@@ -1,20 +1,34 @@
 # About this release
 
-### What is new in this release
+WSO2 Identity Server 5.11.0 is the **latest** WSO2 Identity Server release and is the successor of WSO2 Identity Server 5.10.0.
```
Comment (gomathyk) [inline en/docs/get-started/about-this-release.md:3]: (WSO2 IS)

File: en/docs/get-started/about-this-release.md [hunk 3]
```diff
@@ -59,7 +73,7 @@ release.
 * [5.11.0-M4 Fixes](https://github.com/wso2/product-is/milestone/133?closed=1)
 
 
-### Known issues
+## Known issues
 
 -   For a complete list of open issues related to the WSO2 Identity
```
Comment (gomathyk) [inline en/docs/get-started/about-this-release.md:78]: Is the link incomplete?

**Review by gomathyk — CHANGES_REQUESTED** [review]

File: en/docs/get-started/about-this-release.md [hunk 4]
```diff
@@ -1,20 +1,34 @@
...
+- Introduced CORS management APIs. For more information, see [CORS management APIs](../../develop/cors-rest-api).
+
+- Upgraded to Open SAML 3
+
+
+## Deprecated Features
+
+- /identity/connect/register API
+- SCIM 1.1 API
+- Template Mgt API
```
Comment (GANGANI) [inline en/docs/get-started/about-this-release.md:26]: Though we have deprecated this API, initially we decided to not to announce this. @emswbandara  WDYT of announcing about Template Mgt API deprecation?

Comment (emswbandara) [reply en/docs/get-started/about-this-release.md:26]: this was a hidden feature.. there's no need to announce it.

File: en/docs/get-started/about-this-release.md (hunk 3 above)
Comment (sherenewso2) [reply en/docs/get-started/about-this-release.md]: no, its complete.

**Review by GANGANI — APPROVED** [review]

**Review by NipuniBhagya — APPROVED** [review]

File: en/docs/get-started/about-this-release.md (hunk 1 above)
Comment (emswbandara) [reply en/docs/get-started/about-this-release.md]: Yes, we can have another section for this.

## Features Planned For Deprecation

**Review by gomathyk — APPROVED** [review]

**Pull request merged**